packages/shorebird_cli/lib/src/extensions/arg_results.dart

```diff
@@ -39,3 +41,17 @@ extension OptionFinder on ArgResults {
     return null;
   }
 }
+
+/// Extension on [ArgResults] to provide file validation.
+extension FileValidation on ArgResults {
+  /// Checks if an option is a path that points to an existing file.
+  ///
+  /// This method will only return false when the argument with [name] is
+  /// provided and the file does not exist.
+  bool wasParsedAndFileExists(String name) {
+    final filePath = this[name] as String?;
+    if (filePath == null) return true;
```

Comment:
I'm not sure this is right—if `filePath` is null, that would mean it was not parsed, correct? ArgResults has a `wasParsed` method that might be useful here.